assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -194,6 +194,22 @@ function APIProvider({ children }) {
     [media]
   );
 
+  /**
+   * Delete existing media.
+   *
+   * @param  {number} mediaId
+   * @return {Promise} Media Object Promise.
+   */
+  const deleteMedia = useCallback(
+    (mediaId) => {
+      return apiFetch({
+        path: `${media}/${mediaId}/?force=true`,
```

Comment:
```suggestion
        path: addQueryArgs(`${media}/${mediaId}`, { force: `true` })
```
You need to use `addQueryArgs` here.

Reply:
I don't think there's a _need_ to use that utility here?

Reply:
There is, as if you don't have permalinks enabled, the url pattern looks like this. 
```http://localhost:8899/index.php?rest_route=%2Fwp%2Fv2%2Fweb-story%2F405&context=edit&_locale=user```
Meaning you can't just add `?force=true` without resulting in two ?

Reply:
Or maybe this would be better. 
```
      return apiFetch({
        path: `${media}/${mediaId}`,
        data: { force: true },
        method: 'DELETE',
      });
```

Reply:
What does {force: true} do exactly? I think @pbakaus had some interest in something like this.